### DISCUSSION TITLE
Can i use Code Syntax highlighting like prismjs in outline?

### DISCUSSION THREAD

**Discussion by mingwiki** [outline/outline#4316] [General] [answered]
[Prism](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwja2623z-v6AhVLRd4KHWPfD-YQFnoECAsQAQ&url=https%3A%2F%2Fprismjs.com%2F&usg=AOvVaw2WotHf4bXjbOzD8CJguPuL) or [highlight.js](https://highlightjs.org/) supported?

**Comment by tommoor** [comment, marked as answer]
Code highlighting is supported in code blocks, yes
